examples/tpw-2025-multiple-entries.json

```diff
@@ -0,0 +1,167 @@
+{
+	"entries": [
+		{
+			"metadata": {
+				"scouter": {
+					"name": "omkar",
+					"team": "1072",
+					"app": "tpw"
```

Comment:
for this field (and for other instances of the field across all example files) we changed the app to reference the domain of the website, so it should be `thepurplewarehouse.com` rather than `tpw`